packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -2092,5 +2093,6 @@ Policy Rule conditions aren't supported for this policy.
 | `profileAttributes` | A list of attributes to prompt the user during registration or progressive profiling. Where defined on the User schema, these attributes are persisted in the User profile. Non-schema attributes may also be added, which aren't persisted to the User's profile, but are included in requests to the Registration Inline Hook. A maximum of 10 Profile properties is supported.                                                         | Array | Required | N/A                                                                                                                                                                                                                        |
 | `targetGroupIds`             | (Optional, max 1 entry) The `id` of a Group that this User should be added to                                                     | Array   | No | N/A                                                                                                                                                                                                                         |
 | `unknownUserAction`          | Which action should be taken if this User is new (Valid values: `DENY`, `REGISTER`)                                               | String  | YES | N/A                                                                                                                                                                                                                        |
+| `enrollAuthenticators`       | Additional authenticator fields that should be used on the first page of user registration (Valid values: `password`)             | Array   | No  | N/A |
```

Comment:
Is `password` just the word "password" and it will be displayed as a user registration field, or are we expecting an actual password to be sent via the API? I'm not clear if this is a field name or a value. If it's a field name, should it be `Password` to meet capitalization standards?

Reply:
This is a field name. @dineshkumarjayaraman-okta do you think we should capitalize this?

Reply:
`enrollAuthenticators` is the field. `password` is one of the possible(constant) values. This denotes the type of the authenticators., Eg: password, email, phone, etc. Reference Enum type: `com.okta.services.factors.iface.types.OrgAuthenticatorType`. The `type` field in that enum is all lower case. So, we matched it here.